src/test/java/net/snowflake/client/jdbc/cloud/storage/CloudStorageClientLatestIT.java

```diff
@@ -0,0 +1,47 @@
+package net.snowflake.client.jdbc.cloud.storage;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.fail;
+
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.UUID;
+
+import net.snowflake.client.category.TestCategoryOthers;
+import net.snowflake.client.jdbc.BaseJDBCTest;
+import net.snowflake.client.jdbc.SnowflakeConnection;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+@Category(TestCategoryOthers.class)
+public class CloudStorageClientLatestIT extends BaseJDBCTest {
+
+  /** Test for SNOW-565154 - it was waiting for ~5 minutes */
+  @Test(timeout = 30000L)
```

Comment:
Missing one zero?

Reply:
I want to keep it shorted than 5minuts but not too low so it's 30s